src/index.ts

```diff
@@ -1,14 +1,49 @@
 import * as fs from 'fs';
 import * as path from 'path';
+import * as loadPostCssConfig from 'postcss-load-config';
 import * as ts_module from 'typescript/lib/tsserverlibrary';
+
 import { createMatchers } from './helpers/createMatchers';
 import { isCSSFn } from './helpers/cssExtensions';
 import { getDtsSnapshot } from './helpers/cssSnapshots';
 import { Options } from './options';
 
+import * as postcss from 'postcss';
```

Comment:
We load this in `cssSnapshots.ts`. Was there a reason you decided to load a processor here as well?

Reply:
I decided to move the instantiation of the postcss processor here because it would:

- only be done once
- I needed access for to `info` to call `info.project.getCurrentDirectory()` and didn't want to pass new params into `cssSnapshots.ts`